### DISCUSSION TITLE
Can I use proxy for TLS connection?

### DISCUSSION THREAD

**Discussion by minhdv** [xmppjs/xmpp.js#884] [Q&A]
I saw a fix for websocket connection here https://github.com/xmppjs/xmpp.js/issues/378, but how about TLS connection (xmpps)? In some place,  the application required to use proxy but I don't know how to do.
is there any solution for that?

Thank you in advance

**Comment by sonnyp** [comment]
What kind of proxy? How would you configure it for Node.js ?

I don't think passing `net.Socket` options would help here.

**Comment by arikodcom** [comment]
How I can proxy ip port ? Some one help me ?
